docs/reference/cat/snapshots.asciidoc

```diff
@@ -23,7 +21,7 @@ more repositories. A snapshot is a backup of an index or running {es} cluster.
 +
 --
 (Optional, string) Comma-separated list of snapshot repositories used to limit
```

Comment:
Sorry just missed this: this isn't optional and not a comma separated list either. It's just a mandatory single string (repository name). Could you fix that as well? :)

Reply:
Good catch. Fixed with 022fc9e94d2.

Reply:
>  not a comma separated list either.

Thanks, but this is still missing: It's just the repository name here. No CSV, no wildcards, just the name :)

Reply:
🤦‍♂ Apologies.

Updated with 08bdc76663f.

Reply:
:heart: thanks now it's perfect